go.mod

```diff
@@ -3,21 +3,22 @@ module github.com/tinkerbell/cluster-api-provider-tinkerbell
 go 1.17
 
 require (
-	github.com/go-logr/logr v0.4.0
-	github.com/google/go-cmp v0.5.6
+	github.com/go-logr/logr v1.2.3
+	github.com/google/go-cmp v0.5.8
 	github.com/google/uuid v1.3.0
 	github.com/onsi/gomega v1.19.0
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/pflag v1.0.5
-	github.com/tinkerbell/tink v0.6.1-0.20220419201134-2b877141d755
-	k8s.io/api v0.22.9
-	k8s.io/apimachinery v0.22.9
-	k8s.io/client-go v0.22.9
-	k8s.io/component-base v0.22.9
-	k8s.io/klog/v2 v2.10.0
+	github.com/tinkerbell/rufio v0.0.0-20220601004315-4dc2085adc8e
+	github.com/tinkerbell/tink v0.0.0-20220524234633-0a800a4b5e25
```

Comment:
curious, why did this not use the `v.0.6.1-` prefix here. is this the only way to get the top of tree? do you need top of tree tink library?

Reply:
I think both approaches might work. 
But when i ran my go mod tidy it kept bugging on that and i noticed most of the other dependencies just do v0.0.0 so i kept that.